src/tilegarden/package.json

```diff
@@ -32,7 +32,7 @@
     "local": "yarn build-all-xml && node --inspect=0.0.0.0:9229 -- node_modules/claudia-local-api/bin/claudia-local-api --abbrev 300 --api-module src/api | bunyan -o short",
     "parse-id": "jq -r '.api.id' claudia.json > .api-id",
     "test": "eslint src && jest --coverage",
-    "compile": "yarn build-all-xml && ncc src -o dist && mkdir -p dist/xml && cp src/config/*.xml dist/xml/"
+    "compile": "yarn build-all-xml && mkdir -p dist && cp src/*.js dist && cp -R src/util dist/util && mkdir -p dist/xml && cp src/config/*.xml dist/xml/"
```

Comment:
As @KlaasH and I talked about in our Tilegarden sync, this `ncc` compilation step wasn't really doing anything useful, so I removed it in favor of simply moving the source files to a `dist` directory.